docs/TDD-C-202105-UNL-002.md

```diff
@@ -0,0 +1,92 @@
+# SplitExp
+
+A rule-based method for determining the end of sentence has been developed for Turkish news texts. By including direct quotations that have not been addressed in the problem before, the punctuation ambiguities at the end of the sentence are eliminated by means of a single regular expression.
+
+github repo of dataset: https://github.com/ideateknoloji/SplitExp 
+
+## Dataset Details
+
+This dataset has been created using quotes that are frequently found in Turkish news texts. More than one case was evaluated and a matcher was created over the samples that fit each case. 
```

Comment:
Bu kısımda tam olarak ne kastettiğini anlamadım, tekrar ve daha basit şekilde yazar mısın ? Sanırım alttaki tabloyu anlatmak için paperdaki şu kısıma benzer bir şey yazmaya çalışıyoruz:
```Türkçe haber içeriklerinde cümle sonu belirsizlikleri giderilmesi gereken 4 i¸saret bulunmaktadır: nokta ‘.’, ünlem
i¸sareti ‘!’, soru i¸sareti ‘?’ ve üç nokta karakteri ‘...’ (U+2026).
Çalı¸smamızda önerilen yöntemde, bu karakterlerin cümle sonu
olmayan durumları, alıntılar, sayılar, kısaltmalar ve uzantılar
olmak üzere 4 ba¸slık altında incelenmi¸stir. Bu durumları e¸sleyen düzenli ifadeler, özyineleme ve ¸sartlı yapılar gibi özellikler
kullanılarak olu¸sturulup metni kesintisiz e¸sleyecek tek bir ana
ifadede birle¸stirilmi¸slerdir.```